### PR TITLE
fix(export): eror on empty namespace

### PR DESCRIPTION
**Pull request by sirupsen**
@morgangallant linter went amok too, but whatevs, just look at the test that now passes but didn't before

### CODE REVIEW

File: src/turbopuffer.ts [hunk 1]
```diff
@@ -423,16 +423,16 @@ function toColumnar(vectors: Vector[]): ColumnarVectors {
 }
 
 function fromColumnar(cv: ColumnarVectors): Vector[] {
-  const res = new Array<Vector>(cv.ids.length);
+  const res = new Array<Vector>(cv.ids?.length);
   const attributeEntries = Object.entries(cv.attributes ?? {});
-  for (let i = 0; i < cv.ids.length; i++) {
+  for (let i = 0; i < cv.ids?.length; i++) {
```
Comment (sirupsen) [inline src/turbopuffer.ts:428]: change

File: src/turbopuffer.ts [hunk 2]
```diff
@@ -423,16 +423,16 @@ function toColumnar(vectors: Vector[]): ColumnarVectors {
 }
 
 function fromColumnar(cv: ColumnarVectors): Vector[] {
-  const res = new Array<Vector>(cv.ids.length);
+  const res = new Array<Vector>(cv.ids?.length);
```
Comment (sirupsen) [inline src/turbopuffer.ts:426]: change

File: src/turbopuffer.test.ts [hunk 3]
```diff
@@ -265,3 +282,27 @@ test("connection errors are wrapped", async () => {
     new TurbopufferError("fetch failed: Connect Timeout Error", {}),
   );
 });
+
+test("empty_namespace", async () => {
```
Comment (sirupsen) [inline src/turbopuffer.test.ts:286]: change

File: src/turbopuffer.test.ts [hunk 4]
```diff
@@ -235,10 +253,9 @@ test("sanity", async () => {
     gotError = e;
   }
   expect(gotError).toStrictEqual(
-    new TurbopufferError(
-      "💔 Resource not found; the requested namespace is empty.",
-      { status: 404 },
-    ),
+    new TurbopufferError("🤷 namespace 'typescript_sdk_sanity' was not found", {
```
Comment (sirupsen) [inline src/turbopuffer.test.ts:256]: change, wasn't passing, likely due to your work @morgangallant 👍🏻 better error for sure

Comment (morgangallant) [reply src/turbopuffer.test.ts:256]: thanks for updating this!!

File: src/turbopuffer.ts (hunk 2 above)
Comment (morgangallant) [reply src/turbopuffer.ts:426]: assuming passing undefined to an array constructor just defaults to 0-length array?

**Review by morgangallant — APPROVED** [review]
<3

**Pull request merged**